modules/nw-ovn-kubernetes-list-southbound-database-contents.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-list-ovn-southbound-database-contents_{context}"]
+= Listing the OVN southbound database contents with ovn-kubernetes
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* The OpenShift CLI (oc) installed.
+
+Logic flow rules are stored in the southbound database which is a representation of your infrastructure. Every node in the cluster is a chassis and to view the  southbound database information use `ovn-sbctl show`.
+
+Follow this procedure to list OVN southbound database contents with OVN-Kubernetes.
+
+.Procedure
+
+. Find the OVN raft leader for the southbound database.
```

Comment:
Already showed how to do this in the "Querying the status of the northbound and southbound databases" section.  Do we really want to repeat it?

Reply:
@CFields651 I think I do, as I like sections to stand on their own as much as possible. I guess I feel customers will not read this linearly, and I added to the previous section more as an intro to the table of reference command line arguments. Previous section does not have the full details

Reply:
@CFields651 I guess I feel like each section should stand on its own for the most part, as this is a procedure someone may just go to from the TOC without reading the previous section.